Markdown-to-Word-conversion/Customize-image/.NET-Standard/Customize-image/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.Drawing;
+using System.IO;
+
+namespace Customize_image
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Create a Word document instance.
+            using (WordDocument document = new WordDocument())
+            {
+                //Hook the event to customize the image while importing Markdown.
+                document.MdImportSettings.ImageNodeVisited += MdImportSettings_ImageNodeVisited;
+                //Open the Markdown file.
+                document.Open("Input.md");
+
+                //Create a file stream.
+                using (FileStream outputFileStream = new FileStream(Path.GetFullPath(@"../../../Sample.docx"), FileMode.Create, FileAccess.ReadWrite))
+                {
+                    //Save a Markdown file to the file stream.
+                    document.Save(outputFileStream, FormatType.Docx);
+                }
+            }
+        }
+        private static void MdImportSettings_ImageNodeVisited(object sender, Syncfusion.Office.Markdown.MdImageNodeVisitedEventArgs args)
+        {
+            //Set image stream based on image name from input Markdown.
+            if (args.Uri == "Image_1.png")
+                args.ImageStream = new FileStream("Image_1.png", FileMode.Open);
```

Comment:
Not found image 1 and image 2 files. Could you please ensure the sample is working fine